src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets

```diff
@@ -2,7 +2,7 @@
   
   <PropertyGroup>
     <TodayTimeStamp>$([System.DateTime]::Now.ToString(yyyyMMdd))</TodayTimeStamp>
-    <BuildVersionFilePath>$(BaseIntermediateOutputPath)</BuildVersionFilePath>
+    <BuildVersionFilePath>$(RootBaseIntermediateOutputPath)</BuildVersionFilePath>
```

Comment:
Who is going to set this new property in the other repos that use buildtools?  coreclr, wcf, core-setup?

Also, would it be a terrible idea to use `$(ObjDir)` instead of a new property `$(RootBaseIntermediateOutputPath)`?  Maybe I'm just against the name.... "RootBase" feels redundant/confusing.

Reply:
I don't have a strong opinion here. I  prefer `RootIntermediateOutputPath` (the base in it is a mistake) over `ObjDir` because it's more abstract and does not contain the name of the actual folder in it.

Regards setting the value, I expect follow-up work to be necessary, as in corefx.

Reply:
Giving it a second thought I will revert to ObjDir to minimize necessary changes. It doesn't make sense to refactor now before we move to arcade.

Reply:
That sounds good to me.  Thanks!